<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QListWidget" name="list">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>65</height>
       </size>
      </property>
      <property name="title">
       <string>Disk space</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="targetLabel">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Destination drive:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="targetCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="neededLabel">
         <property name="text">
          <string>Needed:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="availableLabel">
         <property name="text">
          <string>Available:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWrite_image_to_disk"/>
   <addaction name="actionEdit_config"/>
   <addaction name="actionWifi"/>
   <addaction name="actionInfo"/>
   <addaction name="actionBrowser"/>
   <addaction name="actionCancel"/>
  </widget>
  <widget class="QToolBar" name="advToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionWrite_image_to_disk">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/backups.png</normaloff>:/icons/backups.png</iconset>
   </property>
   <property name="text">
    <string>Install (i)</string>
   </property>
   <property name="toolTip">
    <string>Installs the selected operating system image onto this SD card.</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door_in.png</normaloff>:/icons/door_in.png</iconset>
   </property>
   <property name="text">
    <string>Exit (Esc)</string>
   </property>
   <property name="toolTip">
    <string>Exits and boots the installed operating system.</string>
   </property>
   <property name="shortcut">
    <string notr="true">Esc</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/setting_tools.png</normaloff>:/icons/setting_tools.png</iconset>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionEdit_config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/page_white_edit.png</normaloff>:/icons/page_white_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit config (e)</string>
   </property>
   <property name="toolTip">
    <string>Opens the config.txt and cmdline.txt for the installed OS in a text editor.</string>
   </property>
   <property name="shortcut">
    <string notr="true">E</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/world.png</normaloff>:/icons/world.png</iconset>
   </property>
   <property name="text">
    <string>Online help (h)</string>
   </property>
   <property name="toolTip">
    <string>Opens a web browser displaying the Raspberry Pi Forum.</string>
   </property>
   <property name="shortcut">
    <string notr="true">H</string>
   </property>
  </action>
  <action name="actionWifi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/network-wireless.png</normaloff>:/icons/network-wireless.png</iconset>
   </property>
   <property name="text">
    <string>Wifi networks (w)</string>
   </property>
   <property name="toolTip">
    <string>Select a wifi network to connect to</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/information.png</normaloff>:/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>Info (f)</string>
   </property>
   <property name="toolTip">
    <string>Show Information on selected OS</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
